### PR TITLE
Add Youtube explainer Video to about section

### PR DESCRIPTION
**Pull request by maplesyrupsucker**
Revised about section with video

image

### CODE REVIEW

File: css/graphics.css [hunk 1]
```diff
@@ -475,6 +475,12 @@ h1, h2, h3, h4, h5, h6 {
   color:rgba(0,0,0,0.5) !important;
 }
 
+
+.subpage-header .text-inner {
+  display:block !important;
```
Comment (BUYMERCIER) [inline css/graphics.css]: Maybe add spaces here to match with the current coding style:
```css
display: block !important;
```

Comment (BUYMERCIER) [reply css/graphics.css]: Also maybe center the video for small devices
video_align

**Comment by cgcardona** [comment]
@GitCash send .00013092 bch to @maplesyrupsucker

**Comment by cgcardona** [comment]
Testing gitcash.io (built w/ BITBOX)

@GitCash send .00013092 bch to @schancel

**Comment by GitCash** [comment]
[GitCash Tip Response](https://gitcash.io/claim/schancel)
@schancel

**Comment by deepthawtz** [comment]
@GitCash send 100000 satoshis to @maplesyrupsucker dammit

**Comment by GitCash** [comment]
[GitCash Tip Response](https://gitcash.io/claim/maplesyrupsucker)
@maplesyrupsucker

**Pull request merged**